tests/transports-governance/test_helpers.py

```
@@ -0,0 +1,644 @@
+            "max_duration": max(durations),
+            "total_duration": sum(durations),
+        }
+
+    def print_report(self):
+        """Print performance report"""
+        stats = self.get_stats()
+        if stats["count"] == 0:
+            print("No measurements recorded")
+            return
+
+        print(f"\nPerformance Report:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)